### DISCUSSION TITLE
Rendere smart un teleruttore ABB per apertura cancello

### DISCUSSION THREAD

**Discussion by zero0n3** [indomus/forum#3143] [GENERALE - Discussioni generiche]
Salve a tutti.
Ho un teleruttore ABB ESB 20-20 (immagine allegata) installato per azionare il cancello di casa.
L'apertura e chiusura è azionata sia tramite un telecomando remoto via onde radio sia tramite un pulsante posto sulla parete del garage.
Volevo rendere questo teleruttore smart installando un dispositivo tipo Sonoff/Shelly.

Mi sapreste dare un indizio, un modello di dispositivo, uno schema, un qualcosa per poter rendere smart il teleruttore?
Grazie

![image](https://github.com/user-attachments/assets/c0754ad4-12d1-449a-8220-db9df473bc7a)

**Comment by Scialla** [comment]
se in garage ti prende la wifi potresti mettere una qualsiasi dispositivo a contatto pulito (es. shelly 1) sul pulsante configurando l'auto off a 1 secondo.

**Reply by zero0n3** [reply]
ciao @Scialla grazie. Si in garage arriva wifi. Avevo preso inizialmente in questi giorni, uno shelly 1 mini (errore :-( ) perchè pensavo che il pulsante che apre il cancello funzionasse come un pulsante relè quindi con una fase e un neutro. Invece ieri sera aprendo la scatola con i pulsanti, e cercando la fase non la trovavo e ho imparato/scoperto che il pulsante lavora in corrente continua 12V o 24V (dico bene?).
Quindi ora mi doto di uno shelly plus 1 che supporta anche la corrente continua e lo metto in parallelo. 
Sto cercando di capire però, senza avere un cavo di fase ne un neutro nella scatola dei pulsanti, come fare a cablare lo shelly. 
**Per caso hai sottomano uno schema (se si fa) di come dovrei cablare shelly e pulsante?**
Grazie

**Reply by efestopini** [reply]
i pulsante del cancello dovrebbe essere a contatto pulito, non dovresti avere nessuna tensione nel pulsante ed è perfetto lo shelly 1 mini

**Reply by zero0n3** [reply]
grazie @efestopini . A questo punto ti chiedo se hai uno schema sottomano che posso seguire per fare il collegamento. Da quel che capisco lo shelly 1 mini gen 3 non può essere alimentato a 12/24 V ma solo con corrente alternata, ma evidentemente se dici così non ho capito io

**Reply by efestopini** [reply]
lo shelly lo alimenti normalmente a 220 mentre i due fili che ti vanno al pulsante li ponticelli allo 0 e al 1 dello shelly
![Screenshot 2024-07-18 130946](https://github.com/user-attachments/assets/66eff880-bf2e-47f6-932f-a993b2b29a18)

**Reply by zero0n3** [reply]
@efestopini ok..ma se nella scatola dove ho il pulsante, non avessi fili di fase e neutro ma solo i fili 12V che chiudono il contatto col pulsante, per far aprire e chiudere il cancello... sono "obbligato" ad usare uno shelly 1 , giusto?
Oppure dovrei trovare il modo di far arrivare in quella scatola a muro, una fase e un neutro...giusto?

**Reply by efestopini** [reply]
lo shelly può essere alimentato anche in 12v ma come ti dicevo non penso che ci sia il 12v nel pulsante, non so che motore tu abbia ma ma ne ho montati di diverse marche (BFT, Faac, Ditec, DEA system) e in tutti i contatto per l' apertura è sempre stato pulito, se ti arriva il wifi al cancello lo puoi inserire direttamente nel motore, li hai sicuramente la 220, o porti la 220 vicino al pulsante

**Reply by zero0n3** [reply]
@efestopini quindi da totale neofita: 
questo è il pulsante a muro che, premuto, fa scattare il cancello (apri/stop/chiudi/stop) BFT

come si vede, provando col cercafase, in nessuno di entrambi i cavi c'è fase.

**Io prendo lo shelly mini 1 gen3 e collegandolo ai cavi del pulsante, in parallelo, lui funziona?**

(Ovviamente quella pulsantiera è coperta da segnale wifi)

![2](https://github.com/user-attachments/assets/b1287293-37df-46f4-b226-12f8a0a9f43e)
![1](https://github.com/user-attachments/assets/53ac543e-6111-440f-9557-2b2d7b8b174c)

**Reply by Scialla** [reply]
quello di fianco marrone e blu semprerebbe essere una fase e un neutro, dovresti usare un tester però. Lo shelly va bene col contatto pulito ma deve essere alimentato e quindi devi trovare 220v alternata (fase 380 e neutro) oppure 12/24v continua.
Devi usare un tester, il cercafase non basta

**Reply by zero0n3** [reply]
@Scialla che infatti è quello che pensavo di fare in serata appena torno a casa :-D . Poi vi ragguaglio
